Header header
std_msgs/String hand_left
std_msgs/String hand_right
std_msgs/Float64 confid_left
std_msgs/Float64 confid_right